src/testFixtures/kotlin/fixtures/UserFixture.kt

```diff
@@ -0,0 +1,80 @@
+package fixtures
+
+import com.makarytskyi.rentcar.dto.user.CreateUserRequest
+import com.makarytskyi.rentcar.dto.user.UpdateUserRequest
+import com.makarytskyi.rentcar.dto.user.UserResponse
+import com.makarytskyi.rentcar.model.User
+
+object UserFixture {
+    const val userId: String = "1241242"
+    const val createdUserId: String = "83465294"
+
+    val existingUser = User(
```

Comment:
make Fixture to always return new POJO with random values. You can specify value only when it is desired by the Flow, e.g some enum.